src/components/Stake.tsx

```diff
@@ -0,0 +1,44 @@
+import styles from '../styles/Stake.module.css'
+
+export default function Stake({
+  totalStake,
+  totalStakePreviousDay
+}: {
+  totalStake: number
+  totalStakePreviousDay: number
+}) {
+  let delta =
+    ((totalStake - totalStakePreviousDay) / totalStakePreviousDay) * 100
+  const getDelta = (): string => {
+    return `${
+      delta ? (delta > 0 && delta < 0.5 ? '' : delta > 0 ? '+' : '-') : '?'
+    }${delta ? Math.abs(delta).toFixed(0) : ''}%`
+  }
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.container}>
+        <img
+          className={styles.tokenImage}
+          src={'oceanToken.png'}
+          alt="Coin symbol image"
+        />
+        <span className={styles.accuracy}>{totalStake.toFixed(0)}</span>
+      </div>
+      <span
+        className={styles.delta}
+        style={{
+          color: delta
+            ? delta > 0 && delta < 0.5
```

Comment:
We can create a value by using useMemo and we can use it instead of this if-else block.

Reply:
done